### PR TITLE
add node lease namespace

### PR DESCRIPTION
**Pull request by cloudmelon**

### CODE REVIEW

File: content/en/docs/concepts/overview/working-with-objects/namespaces.md [hunk 1]
```diff
@@ -55,13 +55,15 @@ NAME          STATUS    AGE
 default       Active    1d
 kube-system   Active    1d
 kube-public   Active    1d
+kube-node-lease Active  1d
 ```
 
 Kubernetes starts with three initial namespaces:
```
Comment (sftim) [inline content/en/docs/concepts/overview/working-with-objects/namespaces.md:61]: nit:
```suggestion
Kubernetes starts with four initial namespaces:
```

**Comment by cloudmelon** [comment]
@tengqm

Melony QIN

File: content/en/docs/concepts/overview/working-with-objects/namespaces.md (hunk 1 above)
Comment (cloudmelon) [reply content/en/docs/concepts/overview/working-with-objects/namespaces.md:61]: Right, good catch !  :)

File: content/en/docs/concepts/overview/working-with-objects/namespaces.md [hunk 2]
```diff
@@ -55,13 +55,15 @@ NAME          STATUS    AGE
...
 
    * `default` The default namespace for objects with no other namespace
    * `kube-system` The namespace for objects created by the Kubernetes system
    * `kube-public` This namespace is created automatically and is readable by all users (including those not authenticated). This namespace is mostly reserved for cluster usage, in case that some resources should be visible and readable publicly throughout the whole cluster. The public aspect of this namespace is only a convention, not a requirement.
+   * `kube-node-lease` This namespace for the lease objects associated with each node which improves the performance of the node heartbeats as the cluster scales.
```
Comment (tengqm) [inline content/en/docs/concepts/overview/working-with-objects/namespaces.md:66]: This namespace is for ...

**Comment by zacharysarah** [comment]
@cloudmelon 👋 Please rebase this PR and squash your commits.

**Comment by sftim** [comment]
@cloudmelon if you rebase and squash, I _think_ these changes are good to merge.

**Comment by cloudmelon** [comment]
change to another PR

**Pull request closed without merging**